plugins/base/src/test/kotlin/transformers/ContextModuleAndPackageDocumentationReaderTest1.kt

```diff
@@ -110,13 +112,11 @@ class ContextModuleAndPackageDocumentationReaderTest1 : AbstractContextModuleAnd
 
     @Test
     fun `assert moduleA with unknown source set`() {
-        val documentation = reader[
```

Comment:
why this will now throw exception rather than return an empty map?

Reply:
Due to changes in initialization - previously we were passing nulled kotlinAnalyse directly, now it is being retrieved from context and is initialize. And when we try to query it for invalid source set, we receive exception instead.